.github/workflows/build-syncthing.yaml

```diff
@@ -519,6 +526,7 @@ jobs:
           repository: syncthing/release-tools
           path: tools
           fetch-depth: 0
+          ref: ${{ github.ref }} # https://github.com/actions/checkout/issues/882
```

Comment:
As far as I understand that's not needed here (all repos other than syncthing), as tags don't matter?